utils/eslint-config/ruleset/main.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
     }],
     'operator-linebreak': 0,
     'implicit-arrow-linebreak': 0,
+    'no-unused-vars': ['error', {
```

Comment:
Never suddenly change the eslint config without talking to the COP or at least other FE developers, please. Also, this change isn't necessary, since your code above has issues ;)